src/addons/__tests__/Grid.spec.js

```diff
@@ -131,80 +131,80 @@ describe('Grid', () => {
-      component.setState({selectedRows : []});
-    });
-
-    it("should render an additional Select Row column", () => {
-
-      var baseGrid = TestUtils.findRenderedComponentWithType(component, BaseGridStub);
-      var selectRowCol = baseGrid.props.columns[0];
-      expect(baseGrid.props.columns.length).toEqual(columns.length + 1);
-      expect(selectRowCol.key).toEqual('select-row');
-      expect(TestUtils.isElementOfType(selectRowCol.formatter, CheckboxEditorStub)).toBe(true);
-    });
-
-    it("clicking header checkbox should toggle select all rows", () => {
-      //arrange
-      var baseGrid = TestUtils.findRenderedComponentWithType(component, BaseGridStub);
-      var selectRowCol = baseGrid.props.columns[0];
-      var headerCheckbox = selectRowCol.headerRenderer;
-      var checkbox = document.createElement('input');
-      checkbox.type = "checkbox";
-      checkbox.value = "value";
-      checkbox.checked = true;
-      var fakeEvent = {currentTarget : checkbox};
-      //act
-      headerCheckbox.props.onChange(fakeEvent);
-      //assert
-      var selectedRows = component.state.selectedRows;
-      expect(selectedRows.length).toEqual(_rows.length);
-      selectedRows.forEach(function(selected){
-        expect(selected).toBe(true);
-      });
-      //trigger unselect
-      checkbox.checked = false;
-      headerCheckbox.props.onChange(fakeEvent);
-      component.state.selectedRows.forEach(function(selected){
-        expect(selected).toBe(false);
-      });
-    });
-
-    it("should be able to select an individual row when selected = false", () => {
-      component.setState({selectedRows : [false, false, false, false]});
-      var baseGrid = TestUtils.findRenderedComponentWithType(component, BaseGridStub);
-      var selectRowCol = baseGrid.props.columns[0];
-      var fakeEvent = {stopPropagation : function(){}};
-      selectRowCol.onRowSelect(3, fakeEvent);
-      expect(component.state.selectedRows[3]).toBe(true);
-    });
-
-    it("should be able to select an individual row when selected = null", () => {
-      component.setState({selectedRows : [null, null, null, null]});
-      var baseGrid = TestUtils.findRenderedComponentWithType(component, BaseGridStub);
-      var selectRowCol = baseGrid.props.columns[0];
-      var fakeEvent = {stopPropagation : function(){}};
-      selectRowCol.onRowSelect(2, fakeEvent );
-      expect(component.state.selectedRows[2]).toBe(true);
-    });
-
-    it("should be able to unselect an individual row ", () => {
-      component.setState({selectedRows : [null, true, true, true]});
-      var baseGrid = TestUtils.findRenderedComponentWithType(component, BaseGridStub);
-      var selectRowCol = baseGrid.props.columns[0];
-      debugger;
-      var fakeEvent = {stopPropagation : function(){}};
-      selectRowCol.onRowSelect(3, fakeEvent);
-      expect(component.state.selectedRows[3]).toBe(false);
-    });
-  });
-
+  // describe("When row selection enabled", () => {
```

Comment:
ah I think I branched off some of Mo's work. This should be fixed in master. I'll do a catchup